rewrite-java-11/src/main/java/org/openrewrite/java/Java11Parser.java

```diff
@@ -97,10 +98,16 @@ private Java11Parser(@Nullable Collection<Path> classpath,
         // structure of the expression!
         Options.instance(context).put("allowStringFolding", "false");
         Options.instance(context).put("compilePolicy", "check");
+        Options.instance(context).put(Option.G, "");
 
         // JavaCompiler line 452 (call to ImplicitSourcePolicy.decode(..))
         Options.instance(context).put("-implicit", "none");
 
+        //This is a little strange, but by constructing this ahead of the compiler, we are setting the "to do"
+        //instance within the context to a version that is instrumented with micrometer. The compiler will
+        //use this instance by pulling it out of the context.
+        new TimedTodo(context);
```

Comment:
So by passing `context` to this constructor the resulting `TimedTodo` gets associated with that context? Man, that seems so ass-backwards.

Reply:
yeah......the constructor on the super puts itself into the context. Then when the parser say "get the TODO instance", it grabs the instrumented one out rather than creating its own. 

The net effect is the context used by TODO is the same context as the compiler.